src/newcomponents/context/elements/RemoveParticipantLink.js

```diff
@@ -0,0 +1,17 @@
+import ContextLink from "@newcomponents/context/elements/ContextLink";
+
+import { ReactComponent as Remove } from "@icons/context/Remove.svg";
+import conversationService from "@services/conversationsService";
+
+export default function RemoveParticipantLink({ uId }) {
+  return (
+    <ContextLink
+      text="Remove participant"
+      icon={<Remove />}
+      isDangerStyle={true}
+      onClickFunc={async () =>
+        await conversationService.sendRemoveParticipantRequest(uId)
```

Comment:
I propose to do this component less aware of outside world, and pass `onClickFunc ` via prop.

Also onClickFunc -> onClick